docs/using/selecting.md

```diff
@@ -96,7 +96,7 @@ and versioning strategy.
   [cloudpickle](https://github.com/cloudpipe/cloudpickle),
   [dill](https://pypi.org/project/dill/), [numba](https://numba.pydata.org/),
   [bokeh](https://docs.bokeh.org/en/latest/), [sqlalchemy](https://www.sqlalchemy.org/),
-  [hdf5](https://www.h5py.org), [vincent](https://vincent.readthedocs.io/en/latest/),
+  [hdf5](https://www.h5py.org), [altair](https://altair-viz.github.io),
```

Comment:
Vincent is deprecated from 2016-06-18.
It is recommended to install altair, so I replace the library.
https://github.com/wrobstory/vincent#2016-06-18-update

Altair is a nice tool, I've been using it for quite a while.